src/functions/common.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+
+
+# Function to check if the input is a DataFrame and not empty
```

Comment:
I’ve defined the validation in this file since it’s used in both data visualization and spending analysis. 
Please check if the file name and location are appropriate 😊